src/libraries/System.Private.CoreLib/src/System/IO/Path.Windows.cs

```diff
@@ -27,6 +27,39 @@ public static partial class Path
             (char)31
         };
 
+        // Checks if the given path is available for use.
+        public static bool Exists([NotNullWhen(true)] string? path)
+        {
+            try
+            {
+                if (string.IsNullOrEmpty(path))
+                {
+                    return false;
+                }
+
+                string fullPath = GetFullPath(path);
+                Interop.Kernel32.WIN32_FILE_ATTRIBUTE_DATA data = default;
+                int errorCode = FileSystem.FillAttributeInfo(fullPath, ref data, returnErrorOnNotFound: true);
+                bool result = (errorCode == 0) && (data.dwFileAttributes != -1);
```

Comment:
nit: it's better to use `ERROR_SUCCESS`

```suggestion
                bool result = (errorCode == Interop.Errors.ERROR_SUCCESS) && (data.dwFileAttributes != -1);
```

Reply:
Fixed